### PR TITLE
add dev ops page

### PR DESCRIPTION
**Pull request by liuboaibc**
add dev ops page

### CODE REVIEW

File: config/_default/config.toml [hunk 1]
```diff
@@ -46,18 +46,21 @@ hasChildren = true
   [[languages.en.menu.main]]
   parent = "Scenario"
   name = "DevOps"
+  Title = "Embracing One-stop DevOps Workflow"
   URL = "dev-ops"
```
Comment (FeynmanZhou) [inline config/_default/config.toml:50]: Please change the file name and url path from dev-ops to devops.
image

File: content/en/dev-ops/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      contentList:
+        - content: <span>Combine Git with Kubernetes’ convergence, and automates the cloud native Apps delivery</span>
+        - content: <span>Designed for teams, offer built-in multitenancy in DevOps project</span>
+        - content: <span>Liable to be observable,</span> provide dynamic logs for the S2I / B2I build and pipeline
+        - content: Provide audit, alert and notification in pipeline, ensuring quickly locate and solve issues.
+        - content: Support add Git SCM webhooks to trigger a Jenkins build when new commints to the branch
+
+section3:
+  title: See KubeSphere One-stop DevOps Workflow In Action
+  image: /images/service-mesh/15.jpg
```
Comment (FeynmanZhou) [inline content/en/dev-ops/_index.md:45]: All the screenshots in devops page are not matched, you need to verify them with the design layout

image

Comment (liuboaibc) [reply content/en/dev-ops/_index.md:45]: Those pictures can't be downloaded, I have already contacted the UI, temporarily use other pictures instead

Comment (FeynmanZhou) [reply content/en/dev-ops/_index.md:45]: okay, our designer is on vacation today, I guess he might reply to you tomorrow.

**Pull request merged**